Tests/SentryTests/Dynamic/Dynamic.swift

```diff
@@ -0,0 +1,315 @@
+//
+//  Dynamic
+//  Created by Mhd Hejazi on 4/15/20.
+//  Copyright © 2020 Samabox. All rights reserved.
+//
+import Foundation
+
+public typealias ObjC = Dynamic
+
+@dynamicCallable
+@dynamicMemberLookup
+public class Dynamic: CustomDebugStringConvertible, Loggable {
```

Comment:
All classes inside Dynamic folder are auxiliary classes to make it easy to access private properties inside objects for test.